src/styles/blocks/card.scss

```
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 14px;
+  width: 200px;
+  background: $main-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    @extend %flex-center;
+    height: 134px;
+    margin: 15px 5px 45px;
+    transition: transform 0.4s;
+
+    &:hover {
+      box-sizing: content-box;
+      transform: scale(1.2);
+      border: 3px solid #f3f3f3;
+      padding: 12px 0;
+      margin: 0 2px 30px;
+    }
+  }
+
+  &__card-content {
+    @include nav($font-weight-500);
+    line-height: 15px;
+    margin-bottom: 6px;
+  }
+
+  &__card-productCode {
+    margin-bottom: 15px;
+    @include card-productCode($font-weight-400);
+  }
+
+  &__reviews {
+    @extend %flex-space-between;
+    @extend %align-items-center;
+    margin-bottom: 20px;
+
+    &-count {
+      @include card-productCode($font-weight-400);
+    }
+  }
+
+  &__card-price {
+    @extend %flex-space-between;
+    align-items: baseline;
+    margin-bottom: 20px;
+    font-size: 11px;
+  }
+
+  &__card-value {
+    @include card-value($font-weight-700);
+  }
+
+  &__card-btn-link {
+    @extend %flex-center;
+    @extend %align-items-center;
+    @include card-btn-link($font-weight-700);
+    width: 100%;
+    height: 40px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $main-color;
+    background: $color;
+    border-radius: 5px;
+    border: 1px solid $color;
+    transition: 400ms;
```

Always add styles that you apply transition for:
```suggestion
    transition: color 400ms, background 400ms;
```